extensions/vault/runtime/src/main/java/io/quarkus/vault/RuntimeIOException.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.vault;
+
+public class RuntimeIOException extends RuntimeException {
```

Comment:
@vsevel I propose to rename it to `VaultServiceException` or `VaultServiceException`

Reply:
I went for VaultException